crates/rattler_lock/src/parse/models/v6/conda_package_data.rs

```diff
@@ -113,6 +113,10 @@ pub(crate) struct CondaPackageDataModel<'a> {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub input: Option<InputHash<'a>>,
 
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    #[serde_as(as = "Option<PackageBuildSourceSerializer>")]
+    pub package_build_source: Option<PackageBuildSource>,
```

Comment:
Maybe we can think of a better name for this. Just `source` perhaps?

Reply:
I first used `source`, then used `build_source`, and then migrated to `package_build_source` to avoid confusion with `sources`. I also thought about naming it `pixi_build_source`, but didn't want to make it pixi build specific.